crcsim/experiment/prepare.py

```diff
@@ -229,6 +255,61 @@ def create_scenarios() -> List:
         implementation_lower_surveillance.name = f"{fqhc}_implementation_lower_surveillance"
         scenarios.append(implementation_lower_surveillance)
 
+        # Scenarios: 100% FIT compliance
+        implementation_100_compliance = deepcopy(implementation)
+        implementation_100_compliance.transform(
+            transform_repeat_compliance(1.0, "FIT")
+        )
+        implementation_100_compliance.name = f"{fqhc}_implementation_100_compliance"
+        scenarios.append(implementation_100_compliance)
+
+        # Scenarios: 0% FIT Compliance
+        implementation_0_compliance = deepcopy(implementation)
+        implementation_0_compliance.transform(
+            transform_repeat_compliance(0.0, "FIT")
+        )
+        implementation_0_compliance.name = f"{fqhc}_implementation_0_FIT_compliance"
+        scenarios.append(implementation_0_compliance)
+
+        # Scenarios: Delayed Onset Scenarios 50 to 64
+        delayed_onset_50_64 = deepcopy(implementation)
+        delayed_onset_50_64.transform(
+            transform_delayed_onset_compliance(
+                age_ranges=[(50, 64), (65, 75)],
+                compliance_rates=[0.0, 1.0],
+                test="FIT"
+            )
+        )
+        delayed_onset_50_64.name = f"{fqhc}_delayed_onset_50_64"
+        scenarios.append(delayed_onset_50_64)
+
+        # Scenarios: Delayed Onset Scenarios 50 to 59
+        delayed_onset_50_59 = deepcopy(implementation)
+        delayed_onset_50_59.transform(
+            transform_delayed_onset_compliance(
+                age_ranges=[(50, 59), (60, 75)],
+                compliance_rates=[0.0, 1.0],
+                test="FIT"
+            )
+        )
+        delayed_onset_50_59.name = f"{fqhc}_delayed_onset_50_59"
+        scenarios.append(delayed_onset_50_59)
+
+        # Scenario with 50% compliance every year
+        random_50_compliance = deepcopy(implementation)
+        random_50_compliance.transform(
+            transform_random_compliance("FIT", fifty_percent_compliance)
+        )
+        random_50_compliance.name = f"{fqhc}_random_50_compliance"
+        scenarios.append(random_50_compliance)
+
+        # Scenario with 20% compliance every year
+        random_20_compliance = deepcopy(implementation)
+        random_20_compliance.transform(
+            transform_random_compliance("FIT", twenty_percent_compliance)
+        )
+        random_20_compliance.name = f"{fqhc}_random_20_compliance"
+        scenarios.append(random_20_compliance)
```

Comment:
Also on the right track here with the scenarios you've created, just need to adjust the right parameters